themes/bitcore/layout/index.ejs

```diff
@@ -97,7 +97,8 @@
        <a href="https://github.com/streamium/paystream"><li id="streamium-link" class="link"></li></a>
        <a href="https://github.com/dmerrill6/BitBox"><li id="bitbox-link" class="link"></li></a>
        <a href="https://maraoz.github.io/multipaper/"><li id="paper-wallet-link" class="link"></li></a>
-       <a href="http://counterparty.io/"><li id="cp-link" class="link">Counterparty</a></li>
+       <li id="cp-link" class="link"><a href="http://counterparty.io/">Counterparty</a></li>
```

Comment:
this new line has a different format than the previous ones. (instead of <a><li>, <li><a>)